entities/Chef.java

```diff
@@ -0,0 +1,49 @@
+package entities;
+
+import java.util.List;
+
+public class Chef extends Base {
```

Comment:
Feels like `Customer` and `Chef` are pretty much the same, could you abstract these properties to make then re-usable?